sydent/http/servlets/replication.py

```diff
@@ -107,7 +108,9 @@ def _async_render_POST(self, request):
             return
 
         # Process signed associations
-        sg_assocs = inJson.get('sg_assocs', {})
+        sg_assocs = OrderedDict(
+            sorted(inJson.get('sg_assocs', {}).items(), key=lambda k: k[0])
+        )
```

Comment:
Given we just use `sg_assocs.items()` I'd be tempted to just use lists here and below?

Reply:
Also, coooooooooooooment

Reply:
As in, send lists over the wire instead?

Reply:
I mean just do `sg_assocs = list(sorted(...))` instead

Reply:
Ah, I get you. Yes, that is simpler.